_posts/languages/elixir/2000-01-01-start.md

```diff
@@ -31,6 +33,45 @@ A comprehensive list of configuration options is available
 The configuration applied by default is
 [here](https://github.com/HashNuke/heroku-buildpack-elixir/blob/master/elixir_buildpack.config){:target="_blank"}.
 
+{% note %}
+You should always specify your Elixir and OTP target versions. The buildpack
+defaults are several versions back and may not work with your application
+if you are up to date.
+{% endnote %}
+
 ## Phoenix Web Framework
 
-This buildpack let you deploy [Phoenix](https://phoenixframework.org/) applications on Scalingo.
+[Phoenix](https://phoenixframework.org/) support requires combining the
+[elixir buildpack mentionned above](https://github.com/HashNuke/heroku-buildpack-elixir)
+and a [phoenix specific buildpack](https://github.com/gjaldon/heroku-buildpack-phoenix-static):
+the first one handles compiling the Elixir code, the second one handles the assets compilation.
+
+First, remove the `BUILDPACK_URL` from your application's environment.
```

Comment:
```suggestion
```

Reply:
The first part of this page tells the customer to add it. If we leave it, the multi-buildpack is not detected, so it's error-prone not to mention it imo